src/apis/mission.ts

```diff
@@ -45,6 +53,12 @@ const MISSION_APIS = {
     // TODO: data 객체 wrapper 삭제하기 (확인 필요)
     return data.data;
   },
+
+  modifyMission: async (data: ModifyMissionRequest, missionId: number): Promise<ModifyMissionResponse> => {
```

Comment:
![스크린샷 2024-01-11 오전 12 09 10](https://github.com/depromeet/10mm-client-web/assets/68339352/0801c70d-3d1f-4ac8-809b-ce8597bb3757)

ts 에러메세지를 보면 인자가 1개인데 2개를 받았다고 하여 mutationFn 는 argment 가 1개로 제한된 타입을 가지고 있는듯합니다.

 ```suggestion
  modifyMission: async ({ data, missionId }:{
    data: ModifyMissionRequest;
    missionId: number;
  }): Promise<ModifyMissionResponse> => {
```